src/main.cpp

```diff
@@ -394,6 +401,7 @@ void print_help_message() {
   print_option("--exit-fail-on=[CODES]",
                "Fail with a non-zero exit code if any of these");
   print_option("", "errors are found (default: \"all\")");
+  print_option("--s, --stdin", "Read standard input as a javascript file");
```

Comment:
Nit: Capitalize "JavaScript" correctly.
```suggestion
  print_option("--s, --stdin", "Read standard input as a JavaScript file");
```